service/init.go

```diff
@@ -304,6 +310,24 @@ func (pcf *PCF) Exec(c *cli.Context) error {
 	return err
 }
 
+func (pcf *PCF) StartKeepAliveTimer() {
+	ticker := time.NewTicker(30 * time.Second)
+    
+	go func() {
```

Comment:
Should we return function and keep go routing running like this ? Instead isnt it good idea to keep running StartKeepAliveTimer function ?

Reply:
go routine started to handle keepalive timer, changed logic..